socialhome-ganggo.tx-rx.bats

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bats
+#
+# All API calls are documented here: https://ganggo.github.io/api/
+#
+
+load test_helper
+
+endpoint="http://localhost:9001"
+
+@test "$btf create databases" {
+  psql -U postgres -c "create database g1;"
+  [ "$?" -eq 0 ]
+  psql -U postgres -c "create database s1;"
+  [ "$?" -eq 0 ]
+}
+
+@test "$btf start ganggo#1 server" {
+  start_app "g1" "9000" "testing_ganggo:"$(latest_tag "ganggo")
+  [ "$?" -eq 0 ]
+  code=$(wait_for "docker logs g1" "Listening on")
+  echo "expected 0, got $code"
+  [ "$code" -eq "0" ]
+}
+
+@test "$btf start socialhome#1 server" {
+  start_app "s1" "9001" "testing_socialhome" #$(latest_tag "socialhome")
+  [ "$?" -eq 0 ]
+  code=$(wait_for "docker logs s1" "Listening on endpoint tcp")
+  echo "expected 0, got $code"
+  [ "$code" -eq "0" ]
+}
+
+@test "$btf create ganggo#1 user" {
+  post "username=g1&password=pppppp&confirm=pppppp" "http://localhost:9000/users/sign_up"
+  echo "expected 302, got $HTTP_STATUS_CODE"
+  [ "$HTTP_STATUS_CODE" == "302" ]
+}
+
+@test "$btf create socialhome#1 user" {
+  echo "
+from django.contrib.auth import get_user_model;
```

Comment:
```
  File "<string>", line 2
    from django.contrib.auth import get_user_model;
    ^
IndentationError: unexpected indent
```

Also, `;` in Python :fearful: 

(I did also get permission denied to access DB but this indentation is probably also a problem)

Reply:
Though tbh I'm not sure how the indentation is a problem. Maybe cleaner put this snippet in a separate `createuser.py` file and then do `./manage.py shell < createuser.py` which will execute it in django context.